terra/templatetags/terra_extras.py

```diff
@@ -35,3 +35,18 @@ def cap(value):
         # Within 20% of professional development cap
     else:
         return format_currency(value, grouping=True)
+
+
+@register.filter
+def days_cap(value):
+    if value is None or value == "":
+        value = 0
+        return value
+    if value >= 15:
+        return '<span class="alert-danger">{}</span>'.format(value)
+        # Over or at professional development cap
```

Comment:
Minor nitpick: comments normally should go above the relevant code, not below.  No need to change these, just keep in mind for future.

Reply:
Good to know. Thanks!